src/components/Backdrop/Backdrop.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { render, screen } from "@testing-library/react";
+import { userEvent } from "../../utils/test-utils";
+import Backdrop from "./Backdrop";
+
+describe("Backdrop", () => {
+  let show = true;
+
+  function handleClick() {
+    show = false;
+  }
+
+  test("should render backdorp", async () => {
+    render(<Backdrop show={show} onClick={handleClick} />);
+
+    const backdrop = screen.getByRole("dialog");
+    expect(backdrop).toBeInTheDocument();
+  });
+  test("should render handle click", async () => {
+    render(<Backdrop show={show} onClick={handleClick} />);
+
+    const backdrop = screen.getByRole("dialog");
+
+    await userEvent.click(backdrop);
+    expect(show).toBe(false);
```

Comment:
Creo que se debería chequear que aparte de que la variable show este en false, que realmente no se muestre el backdrop.

```suggestion
    expect(show).toBe(false);
    expect(backdrop).not.toBeInTheDocument()
```

Reply:
añadí un test que chequea si tiene las clases correspondientes cuando no se muestra